src/bubbleSort.js

```diff
@@ -0,0 +1,24 @@
+export default function bubbleSort(array){
```

Comment:
I tried testing this code and it didn't work. I used this website repl.it and most cases code like this should be able to run well. Sorry I don't have the knowledge to exactly pinpoint which part of the code isn't working.